### DISCUSSION TITLE
Alerting about a prediction should be possible.

### DISCUSSION THREAD

**Discussion by j-dc** [grafana/grafana#38748] [Alerting]
We use grafana to show predictions.

We want to send an alert if a certain prediction is higher than a certain value.

At this time (v8.1.2)   I can only use (for example) this: 
    
    query (A, 12h, now)

meaning:  the alert will check data from the last 12h.

What I want to do is something like this:

    query(A, now, 12h)

meaning:   the alert will check data for the next 12h.

I have had no previous issues with showing predictions, as your date-time selection can perfectly show the data for the next 12hours.
I would be delighted if this would be possible for the alerts.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
